pkg/plugins/runtime/gateway/gateway_route_generator_test.go

```diff
@@ -1606,13 +1597,7 @@ conf:
       - destination:
           kuma.io/service: echo-service
 `,
-			WithoutResource{
-				Resource: core_mesh.TimeoutType,
-				Mesh:     "default",
-				Name:     "timeout-all-default",
```

Comment:
How exactly should these tests be changing, don't we kind of need both cases? So for example here we now need to _add_ `Timeout` resources?

Reply:
Reverted change, I am going to switch in the next PR and add default policy